lib/core/versions/latest/models/ChunkFileModel.ts

```diff
@@ -2,5 +2,5 @@
  * Defines the external Chunk File structure.
  */
 export default interface ChunkFileModel {
-  deltas: string[];
+  deltas: object[];
```

Comment:
this is weak typing, prefer exact type.

Reply:
It is intentionally object because we don't know for sure the deltas are valid delta objects yet, we just know they need to be objects (similar to before we didn't know if they were valid base64encoded strings that can decode into delta, we just know they are strings). There are more checks later on in the process that validates they are delta objects during resolution time.

Reply:
I see, that makes sense... a comment to this effect may be helpful to future readers.

Reply:
Note: We have internal discussion to see if we want to check the type during observing time, we are worried about the performance, but if we decide to do it, it can easily be versioned in.

Reply:
hmm yep, wondering what happens when its bad... prefer to throw stuff out as soon as possible if it's bad. may want to link to an issue for future discussion related to this.

Reply:
Ok, will make an issue for it and add comment in code.

Reply:
https://github.com/decentralized-identity/sidetree/issues/880